frontend/src/store/modules/geo_location/actions.js

```diff
@@ -1,11 +1,37 @@
-import {CHANGE_SELECTED_PERIOD, CHANGE_SELECTED_PARAMETER} from "./types/actions";
-import {SET_SELECTED_PERIOD, SET_SELECTED_PARAMETER} from "./types/mutations";
+import {
+    CHANGE_SELECTED_PERIOD,
+    CHANGE_SELECTED_PARAMETER,
+    FETCH_GEO_LOCATION_ITEMS
+} from "./types/actions";
+import {
+    SET_SELECTED_PERIOD,
+    SET_SELECTED_PARAMETER,
+    SET_GEO_LOCATION_ITEMS,
+    SET_IS_FETCHING,
+    RESET_IS_FETCHING
+} from "./types/mutations";
+import {getGeoLocationItems} from '@/api/geo_location/geoLocationItemsService';
+import {getTimeByPeriod} from '@/services/periodService';
 
 export default {
     [CHANGE_SELECTED_PERIOD]: (context, payload) => {
         context.commit(SET_SELECTED_PERIOD, payload.value);
+        context.commit(SET_IS_FETCHING);
```

Comment:
it's better to make this mutation in fetch action, since you resetting it there, when fetch is completed